packages/connectivity/connectivity_for_web/lib/src/utils/connectivity_result.dart

```diff
@@ -1,27 +1,22 @@
+import 'dart:html' as html show NetworkInformation;
 import 'package:connectivity_platform_interface/connectivity_platform_interface.dart';
 
 /// Converts an incoming NetworkInformation object into the correct ConnectivityResult.
-//
-// We can't be more specific on the signature of this method because the API is odd,
-// data can come from a static value in the DOM, or as the 'target' of a DOM Event.
-//
-// If we type info as `NetworkInformation`, Dart will complain with:
-// "Uncaught Error: Expected a value of type 'NetworkInformation',
-// but got one of type 'NetworkInformation'"
```

Comment:
This was probably Dart telling me that there already existed a native `NetworkInformation` class, but I misinterpreted as "coming from different parts"